src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -158,7 +158,7 @@ namespace AirLoopHVACDOAS {
                 AirLoopMixer thisMixer;
 
                 thisMixer.name = UtilityRoutines::MakeUPPERCase(thisObjectName);
-                thisMixer.OutletNodeName = UtilityRoutines::MakeUPPERCase(fields.at("outlet_node_name"));
+                thisMixer.OutletNodeName = UtilityRoutines::MakeUPPERCase(AsString(fields.at("outlet_node_name")));
```

Comment:
OK, so we should use `AsString` to get a string when it is needed as an argument.  And the idea is that these worker routines should more and more take string views themselves so that this intermediary can be stripped out, making the code better as we go along.